rewrite-gradle/src/main/resources/META-INF/rewrite/gradle-8.yml

```diff
@@ -24,6 +24,9 @@ recipeList:
       version: 8.x
       addIfMissing: false
 
+  # https://docs.gradle.org/8.11.1/userguide/upgrading_version_8.html#test_framework_implementation_dependencies
+  - org.openrewrite.gradle.AddJUnitPlatformLauncher
```

Comment:
optional: Is there a benefit in the indirection here rather than the AddDependency being here directly?

Reply:
Only in the sense that it becomes a searchablle migration step, that folks can run separately as they are not likely to re-run `MigrateToGradle8` when already on Gradle 8. It is indeed optional.

Reply:
Fair enough. One thing that I've wondered is how to approach these sort of utility recipes as searching for them in the recipe catalog can become pretty overwhelming, but that's an entirely separate problem.